### ISSUE TITLE
[Feature Request] Automatic Text centering 

### ISSUE THREAD

**Issue by Piotr-Niewczas** [jmanpenilla/MiniMOTD#33]
Hi,
Adding spaces to center text is quite annoying. I'd love to see an option for the plugin to automatically center text within the given line.

Example:
- "Text on the left <center>Text in the middle</center> text on the end{br}<center> Text in the middle of the next line</center>"
![image](https://user-images.githubusercontent.com/74670892/101286613-bfd23a00-37eb-11eb-8968-1ff673d8c719.png)

**Comment by Kale-Ko** [comment]
Yes, very good idea!

**Comment by pandather** [comment]
i think we would want a <middle> operator that, if possible, fully centers the message.

**Comment by KyleJKC** [comment]
This is a very good idea.

**Comment by LuciusOfLegend** [comment]
Please implement this, it would be great!

**Comment by pomonella01** [comment]
+1 for this. will there be any issues merging it?

**Comment by FQQD** [comment]
Any progress on this? I'd love to see it.

**Comment by SergSel2006** [comment]
+1 here, would like to use it!

**Comment by quiquelhappy** [comment]
+1

**Comment by PavelGuet** [comment]
+1

**Comment by MnLand** [comment]
+1

**Comment by Slymeball** [comment]
+1 on this, I could see this being used in chat and MOTDs.

**Comment by Nukecraft5419** [comment]
Is there any word on whether this feature is in development or has been put aside, I would recommend in addition to that one more thing namely conversion of HEX colors to legacy for clients below 1.16

**Comment by mudkipdev** [comment]
> Is there any word on whether this feature is in development or has been put aside, I would recommend in addition to that one more thing namely conversion of HEX colors to legacy for clients below 1.16

@Nukecraft5419 Color downsampling for 1.15 clients is already a feature.

**Comment by Nukecraft5419** [comment]
> > Is there any word on whether this feature is in development or has been put aside, I would recommend in addition to that one more thing namely conversion of HEX colors to legacy for clients below 1.16
> 
> @Nukecraft5419 Color downsampling for 1.15 clients is already a feature.

Yes in fact I did a search shortly after I noticed that it was already there

**Comment by salocinyeet** [comment]
+1
I would really like this

**Comment by GegTheDev** [comment]
Pleaseee

**Comment by abiriadev** [comment]
The thing is, aligning text is not supported by [minimessage format](https://docs.advntr.dev/minimessage/format.html) used by this plugin.

**Comment by Kreateo** [comment]
Looking for this as well - I'm actually surprised it hasn't already been implemented considering the **want** for this feature.

**Comment by kylemarham** [comment]
+1 @mudkipdev

**Comment by SergSel2006** [comment]
I am very sad and surprised that we still have this open after 5 years.

**Comment by benjith17** [comment]
Just found this thread while googling if it was possible. Please add its so useful for so many things!

**Comment by PageFault99** [comment]
+1

**Comment by larryshow0213** [comment]
+1

**Comment by Nixoly** [comment]
+1

**Comment by itzzjustmateo** [comment]
+99 xd

**Comment by WinsomeGuy** [comment]
+100 ( are we ever getting this )

**Comment by itzzjustmateo** [comment]
@WinsomeGuy Maybe, but just maybe, when I am bored, I can implement this, by contributing
